src/core/server/logging/logging_config.ts

```diff
@@ -65,7 +65,13 @@ export const config = {
       defaultValue: new Map<string, AppenderConfigType>(),
     }),
     loggers: schema.arrayOf(loggerSchema, {
-      defaultValue: [],
+      defaultValue: [
```

Comment:
realised that configuring any loggers would remove this default and thereby switch the `elasticsearch.deprecation` logger level to `info`...

So I need to have logic that's a bit cleverer

Reply:
Unless we decide to enable this logging context by default, might be useful information to see how many logs get triggered on Cloud

Reply:
Yeah, what you are running into here is somewhat related to this enhancement issue we have for default appender configs: https://github.com/elastic/kibana/issues/92082

What would be useful here is to be able to do something similar for loggers.

> Unless we decide to enable this logging context by default, might be useful information to see how many logs get triggered on Cloud

FWIW I would be in support of this, but if we went down this path I would vote that we clean up the structure of the log messages just a bit (see my other comment)

Reply:
You chose the hard path by not just using the `debug` log level 😅 

Even if I think this approach to use more normal log levels such as `warning` and `error`, and try to disable the `elasticsearch.deprecation` by default is nice, what is the main reason to not use `debug` instead?
- It requires a lot less changes
- It's more consistent with the other 'disabled by default' logger we currently have, such as `elasticsearch.query` or `http.server.response`